dev/ProjectReunion_DLL/ProjectReunion_DLL.vcxproj

```diff
@@ -167,7 +168,7 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <ClCompile>
       <PrecompiledHeader>Use</PrecompiledHeader>
-      <WarningLevel>Level3</WarningLevel>
+      <WarningLevel>Level4</WarningLevel>
```

Comment:
Thanks!

Reply:
I /Wall but got oodles of warnings about things we use. I plan to follow up with WIL, Detours, C++/WinRT and Windows teams later. For now, /W4 is as good as we can get. We can selectively enable more if we find some common or egregious issues.